ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,52 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+        & "elasticsearch"
+    """
+    namespace_obj1 = ocp.OCP(
```

Comment:
I think these var names are still too vague. `obj1` means nothing to someone reading the code below. I know that they are namespaces but I don't know why obj1 is different than obj2.

`openshift_logging_namesapce` or `namespace_openshift_logging` immediately tells them that it is the openshift-logging namespace. Figuring out what a variable represents shouldn't take tracing it back to it's instantiation.